react/src/components/footer/SprkFooter.js

```diff
@@ -137,18 +158,34 @@ class SprkFooter extends Component {
               <SprkFooterAwards awards={awards} />
             )}
 
-            {paragraphs.length > 0
-              && paragraphsHasIds.map(p => (
+            {paragraphs.length > 0 &&
+              paragraphsHasIds.map((p) => (
                 <p
                   key={p.id}
-                  className="sprk-o-Stack__item sprk-b-TypeBodyFour sprk-c-Footer__text"
+                  className="
+                    sprk-o-Stack__item
+                    sprk-b-TypeBodyFour
+                    sprk-c-Footer__text
+                  "
                 >
                   {p.text}
                 </p>
               ))}
 
+            {/* To allow users to use inline links until
+            compositional footer is available */}
+            {additionalDisclaimer}
```

Comment:
This is where the additional disclaimer text is being rendered (calling it out because there's a bunch of formatting stuff)